.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@
 #based on https://raw.githubusercontent.com/ckan/ckanext-scheming/master/.github/workflows/test.yml
 # alternative https://github.com/ckan/ckan/blob/master/contrib/cookiecutter/ckan_extension/%7B%7Bcookiecutter.project%7D%7D/.github/workflows/test.yml
 name: Tests
-on: [push, pull_request]
+on: [push]
```

Comment:
why remove pull_request?

Reply:
push does not run when its external of our repo, also if there is substantial differences between main branch and merging branch, it might have passed on push, but failed on integration merged code. I have seen this occur lots of times on the foodpantry work, primarily due to flake8 rules being updated/enabled.